### DISCUSSION TITLE
Feature Request - Cybereason Elastic Integration

### DISCUSSION THREAD

**Discussion by brazorfajeje** [Security-Onion-Solutions/securityonion#13753] [Unsupported Versions]
Is it possible to add integration support for cybereason (https://www.elastic.co/docs/current/integrations/cybereason) ?

Thanks

**Comment by cm-ops** [comment]
I'll pass this request on.

**Comment by brazorfajeje** [comment]
in case the team does not take charge of adding support for this integration, is it possible to install integrations that are not officially supported?
